src/Umbraco.Tests.UnitTests/Umbraco.Core/Models/Collections/PropertyCollectionTests.cs

```diff
@@ -3,32 +3,33 @@
 using NUnit.Framework;
 using Umbraco.Core;
 using Umbraco.Core.Models;
+using Umbraco.Tests.Common.Builders;
 using Umbraco.Tests.TestHelpers;
-using Umbraco.Tests.TestHelpers.Entities;
-using Umbraco.Tests.Testing;
 
-namespace Umbraco.Tests.Models.Collections
+namespace Umbraco.Tests.UnitTests.Umbraco.Core.Modelsl.Collections
```

Comment:
Looks like a mistake

```suggestion
namespace Umbraco.Tests.UnitTests.Umbraco.Core.Models.Collections
```